src/EnergyPlus/HeatBalanceSurfaceManager.hh

```diff
@@ -55,6 +55,7 @@
 #include <EnergyPlus/Data/BaseData.hh>
 #include <EnergyPlus/DataWindowEquivalentLayer.hh>
 #include <EnergyPlus/EnergyPlus.hh>
+#include <EnergyPlus/Material.hh>
```

Comment:
Should not need this if you eliminate the "helper" functions.

Reply:
removed it